deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -135,29 +135,36 @@ def compute(self, context: Context) -> CheckResult:
         """
         values_dict = OrderedDict()
         displays_dict = OrderedDict()
-        label_measures_names = [x['name'] for x in self._label_measurements]
-        for label_measure in self._label_measurements:
-            measure_name = label_measure['name']
+        label_properties_names = [x['name'] for x in self._label_properties]
+        for label_property in self._label_properties:
+            name = label_property['name']
+            value_type = label_property['value_type']
+            # If type is class converts to label names
+            if value_type == 'class_id':
+                self._train_label_properties[name] = [context.train.label_id_to_name(class_id) for class_id in
+                                                      self._train_label_properties[name]]
+                self._test_label_properties[name] = [context.test.label_id_to_name(class_id) for class_id in
+                                                     self._test_label_properties[name]]
 
             value, method, display = calc_drift_and_plot(
-                train_column=pd.Series(self._train_label_properties[measure_name]),
-                test_column=pd.Series(self._test_label_properties[measure_name]),
-                plot_title=measure_name,
-                column_type='numerical' if label_measure['is_continuous'] else 'categorical',
+                train_column=pd.Series(self._train_label_properties[name]),
+                test_column=pd.Series(self._test_label_properties[name]),
+                plot_title=name,
+                column_type='categorical' if value_type == 'class_id' else value_type,
```

Comment:
Didn't you say there's "class_id", "continuous" and "discrete"?

Reply:
BTW maybe it should be "output_type"

Reply:
I said numerical/categorical. Doesn't care much, but this are the names from tabular